app/views/contracts/index.html.erb

```diff
@@ -25,6 +33,14 @@
         <td>
           <%= number_to_currency(contract.full_reports.sum(:cost), unit: '€') %>
         </td>
+        <td>
+          <%= form_with(model: contract, method: :patch, local: true) do |form| %>
+            <div class="form-group">
+              <%= form.hidden_field(:aasm_state, value: contract.next_state) %>
+              <%= form.submit contract.next_event, class: "btn btn-primary" %>
```

Comment:
One quick thing, shall we use different colours for the buttons, so that it's a bit more distinctive, so that that they don't all look the same.

Reply:
Do you mean by status? Any example? I think I can use a helper for this, right?

Reply:
Yes by status, like:

`btn-primary` to move from proposal -> to live;
`btn-success` to move from live -> to finish;
`btn-warning` to move from finished -> to archive.

yeah a helper would work.